src/main/kotlin/com/example/todo/Service.kt

```diff
@@ -29,4 +31,10 @@ class SimpleTodoService (
 
         return todo.toTodoDTO()
     }
+
+    override fun toDone(id: Long): TodoDTO{
+        val todo: Todo = todoRepository.findByIdOrNull(id) ?: throw IllegalArgumentException()
```

Comment:
P3: optional 클래스에서는 `orElseThrow`를 제공해줍니다. 예외타입은 `NoSuchElementException`입니다.
```suggestion
        val todo: Todo = todoRepository.findById(id).orElseThrow()
```

Reply:
https://github.com/jimin0127/kotlin-spring-todo/pull/3/commits/caa0fa07ad85f58a3fef7585e8cfc2523f83b894
반영했습니다. 그런데 예외처리를 어떻게 해야할까요..? try-catch 문으로 처리하려고 했는데 그렇게 되면 try-catch문 바깥쪽에 어떤 값을 반환해야할지 모르겠어요...

Reply:
Spring MVC에는`Exception Handler`가 있습니다. 예외가 발생하면 별도로 처리해 주어야 할 필요가 없다면 되도록 예외를 던져서 handler가 처리하도록 하는게 좋아요.

https://spring.io/blog/2013/11/01/exception-handling-in-spring-mvc

Reply:
이 부분은 따로 반영해보고 따로 반영해보도록 하겠습니다! 일단 머지 할게여..!